Add nvim_open_win configuration for file-popup action

FollieHiyuki opened the pull request:
Judging by https://github.com/kyazdani42/nvim-tree.lua/pull/1042#discussion_r819384771 maybe I should put the configuration inside `actions.file_popup.nvim_open_win` instead.

alex-courtis reviewed (changes requested):
Looks good, works.

> Judging by [#1042 (comment)](https://github.com/kyazdani42/nvim-tree.lua/pull/1042#discussion_r819384771) maybe I should put the configuration inside `actions.file_popup.nvim_open_win` instead.

Yes please, your timing is very good.

- [ ] Follow #1377 to allow all `nvim_open_win` options
- [ ] Add `file_popup.open_win_config` to help
- [ ] Update help as per [CONTRIBUTING.md](https://github.com/kyazdani42/nvim-tree.lua/blob/master/CONTRIBUTING.md)

FollieHiyuki commented:
I updated the help page and moved the config to `actions.file_popup.open_win_config`. The variable was renamed to reflect the change.

`open_win_config` is already in FIELD_SKIP_VALIDATE so I don't think I need to do anything extra.
